src/script/util/Entropy.test.ts

```diff
@@ -92,5 +131,348 @@ describe('Entropy', () => {
     it('returns zero for empty entropyData', () => {
       expect(new EntropyData().entropyBits).toEqual(0);
     });
+
+    it('makes sense with no movement', () => {
+      const data = new EntropyData();
+      [...Array(300)].forEach((_, i) => {
+        data.addFrame({t: 0, x: 0, y: 0});
```

Comment:
i have a general question about `t` in all of the subsequent tests: 

shouldn't it increment sequentially for testing appropriately? 

seems it is throwing off our final entropy values if we set it to be zero when this *should* never happen in a real life use case. 

```suggestion
      [...Array(300)].forEach((_, i) => {
        data.addFrame({t: i, x: 0, y: 0});
```

Reply:
The idea here is to provide the worst possible values for `t` and that is the same value for all entries. This then shows that the resulting entropy value is only based on `x` and `y` but not `t`